### PR TITLE
[GameInformer] Fix brightcove_id regex.

### PR DESCRIPTION
**Pull request by pongopeter826**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [ x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [ x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [ x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [ x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.
Gameinformer has updated it's website and the brightcove id is not in the same place I have updated the regex that searches for it to reflect these changes

### CODE REVIEW

File: youtube_dl/extractor/gameinformer.py [hunk 1]
```diff
@@ -24,5 +24,5 @@ class GameInformerIE(InfoExtractor):
     def _real_extract(self, url):
         display_id = self._match_id(url)
         webpage = self._download_webpage(url, display_id)
-        brightcove_id = self._search_regex(r"getVideo\('[^']+video_id=(\d+)", webpage, 'brightcove id')
+        brightcove_id = self._search_regex(r"<p><div id=\"bc_(\d+)", webpage, 'brightcove id')
```
Comment (dstftw) [inline youtube_dl/extractor/gameinformer.py]: Keep the old regex as well.

Comment (pongopeter826) [reply youtube_dl/extractor/gameinformer.py]: Ok, But why would I need to do that if the current one is working?

**Comment by pongopeter826** [comment]
Is this ok?

File: youtube_dl/extractor/gameinformer.py [hunk 2]
```diff
@@ -24,5 +24,7 @@ class GameInformerIE(InfoExtractor):
     def _real_extract(self, url):
         display_id = self._match_id(url)
         webpage = self._download_webpage(url, display_id)
-        brightcove_id = self._search_regex(r"getVideo\('[^']+video_id=(\d+)", webpage, 'brightcove id')
+        regex = ["getVideo\('[^']+video_id=(\d+)", "<p><div id=\"bc_(\d+)"]
+        brightcove_id = self._search_regex(regex, webpage, 'brightcove id')
         return self.url_result(self.BRIGHTCOVE_URL_TEMPLATE % brightcove_id, 'BrightcoveNew', brightcove_id)
+
```
Comment (dstftw) [inline youtube_dl/extractor/gameinformer.py]: Remove this.

File: youtube_dl/extractor/gameinformer.py [hunk 3]
```diff
@@ -24,5 +24,7 @@ class GameInformerIE(InfoExtractor):
     def _real_extract(self, url):
         display_id = self._match_id(url)
         webpage = self._download_webpage(url, display_id)
-        brightcove_id = self._search_regex(r"getVideo\('[^']+video_id=(\d+)", webpage, 'brightcove id')
+        regex = ["getVideo\('[^']+video_id=(\d+)", "<p><div id=\"bc_(\d+)"]
```
Comment (dstftw) [inline youtube_dl/extractor/gameinformer.py]: Inline it into actual method call.

File: youtube_dl/extractor/gameinformer.py [hunk 4]
```diff
@@ -24,5 +24,5 @@ class GameInformerIE(InfoExtractor):
     def _real_extract(self, url):
         display_id = self._match_id(url)
         webpage = self._download_webpage(url, display_id)
-        brightcove_id = self._search_regex(r"getVideo\('[^']+video_id=(\d+)", webpage, 'brightcove id')
+        brightcove_id = self._search_regex(["getVideo\('[^']+video_id=(\d+)", "<p><div id=\"bc_(\d+)"], webpage, 'brightcove id')
```
Comment (dstftw) [inline youtube_dl/extractor/gameinformer.py]: Do not remove `r` prefix.

File: youtube_dl/extractor/gameinformer.py [hunk 5]
```diff
@@ -24,5 +24,6 @@ class GameInformerIE(InfoExtractor):
     def _real_extract(self, url):
         display_id = self._match_id(url)
         webpage = self._download_webpage(url, display_id)
-        brightcove_id = self._search_regex(r"getVideo\('[^']+video_id=(\d+)", webpage, 'brightcove id')
+        brightcove_id = self._search_regex([r"getVideo\('[^']+video_id=(\d+)", r"<p><div id=\"bc_(\d+)"], webpage, 'brightcove id')
         return self.url_result(self.BRIGHTCOVE_URL_TEMPLATE % brightcove_id, 'BrightcoveNew', brightcove_id)
+
```
Comment (dstftw) [inline youtube_dl/extractor/gameinformer.py]: You must be kidding me?

Comment (pongopeter826) [reply youtube_dl/extractor/gameinformer.py]: Sorry I didn't see that extra space.

**Pull request merged**